components/articles/articleAuthor.tsx

```diff
@@ -20,7 +20,15 @@ const ArticleAuthor = ({ name, position, image }: ArticleAuthorProps) => {
           className="size-10 rounded-full object-cover"
         />
       )}
-      <div className="font-semibold uppercase">{name}</div>
+      <div className="font-semibold uppercase">
+        {url ? (
+          <a className="no-underline" href={url}>
+            {name}
+          </a>
```

Comment:
We're on NextJs, so I would always try to use `<Link>`
```suggestion
          <Link className="no-underline" href={url}>
            {name}
          </ Link>
```